mlkem/kem.h

```diff
@@ -13,7 +13,12 @@
 #if defined(MLK_CHECK_APIS)
 /* Include to ensure consistency between internal kem.h
  * and external mlkem_native.h. */
+#if defined(MLK_CONFIG_API_NO_SUPERCOP)
+#error Unexpected configuration of MLK_CONFIG_API_XXX during build
```

Comment:
```suggestion
#error Unexpected configuration of MLK_CONFIG_API_NO_SUPERCOP during build
```

Reply:
Removing. This is checked in `common.h` already.